### PR TITLE
Removed unnecessary inheritDoc.

### PR DESCRIPTION
**Pull request by VelinYordanov**
See openhab/static-code-analysis#103

Signed-off-by: VelinYordanov <velin.iordanov@gmail.com>

### CODE REVIEW

File: addons/binding/org.openhab.binding.minecraft/src/main/java/org/openhab/binding/minecraft/internal/discovery/MinecraftDiscoveryService.java [hunk 1]
```diff
@@ -157,11 +150,8 @@ private void submitPlayerDiscoveryResults(ThingUID bridgeUID, String name) {
 
         Map<String, Object> properties = new HashMap<>();
         properties.put(MinecraftBindingConstants.PARAMETER_PLAYER_NAME, name);
-        thingDiscovered(DiscoveryResultBuilder.create(uid)
-                .withProperties(properties)
-                .withBridge(bridgeUID)
-                .withLabel(name)
-                .build());
+        thingDiscovered(DiscoveryResultBuilder.create(uid).withProperties(properties).withBridge(bridgeUID)
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.minecraft/src/main/java/org/openhab/binding/minecraft/internal/discovery/MinecraftDiscoveryService.java]: Pitty

File: addons/binding/org.openhab.binding.pulseaudio/src/main/java/org/openhab/binding/pulseaudio/handler/PulseaudioHandler.java [hunk 2]
```diff
@@ -249,8 +246,10 @@ public void onDeviceStateChanged(ThingUID bridge, AbstractAudioDeviceConfig devi
             updateState(PulseaudioBindingConstants.STATE_CHANNEL,
                     device.getState() != null ? new StringType(device.getState().toString()) : new StringType("-"));
             if (device instanceof SinkInput) {
-                updateState(PulseaudioBindingConstants.ROUTE_TO_SINK_CHANNEL, ((SinkInput) device).getSink() != null
-                        ? new StringType(((SinkInput) device).getSink().getPaName()) : new StringType("-"));
+                updateState(PulseaudioBindingConstants.ROUTE_TO_SINK_CHANNEL,
+                        ((SinkInput) device).getSink() != null
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.pulseaudio/src/main/java/org/openhab/binding/pulseaudio/handler/PulseaudioHandler.java]: Why does the formatter break builders and improves this ... :-)

File: addons/binding/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/protocol/xml/InputWithDabControlXML.java [hunk 3]
```diff
@@ -97,71 +99,71 @@ public void update() throws IOException, ReceivedMessageParseException {
             throw new ReceivedMessageParseException("<Play_Info>GetParam failed: " + response);
         }
 
-        //Sample response:
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/protocol/xml/InputWithDabControlXML.java]: This should be reverted, can you guard it using an `// @formatter:off ` and `// @formatter:on`

File: addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxDevicesHandler.java [hunk 4]
```diff
@@ -551,8 +545,12 @@ private void setProperties(Device device) {
         }
     }
 
-    /* (non-Javadoc)
-     * @see org.eclipse.smarthome.core.thing.binding.BaseThingHandler#bridgeStatusChanged(org.eclipse.smarthome.core.thing.ThingStatusInfo)
+    /*
+     * (non-Javadoc)
+     * 
+     * @see
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxDevicesHandler.java]: I suppose this one should be removed as well.

@kaikreuzer we have 18 `(non-Javadoc)` docblock like things, for example this one and the following one:

```
    /*
     * (non-Javadoc)
     *
     * @see java.lang.Object#toString()
     */
```

Should this be removed and maybe added as a check as well

See also https://stackoverflow.com/questions/4597329/does-non-javadoc-have-a-well-understood-meaning

**Comment by VelinYordanov** [comment]
Thanks for the review. I updated the pull request.

**Pull request merged**